projects/netgrif-components/src/lib/data-fields/i18n-field/i18n-divider-field/i18n-divider-field.component.spec.ts

```diff
@@ -1,21 +1,41 @@
 import {ComponentFixture, TestBed} from '@angular/core/testing';
 import {I18nDividerFieldComponent} from './i18n-divider-field.component';
-import {Component} from '@angular/core';
+import {Component, CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
 import {FormControl} from '@angular/forms';
-import {I18nField, WrappedBoolean} from '@netgrif/components-core';
+import {
+    AuthenticationMethodService, AuthenticationService, ConfigurationService,
+    I18nField,
+    MaterialModule,
+    MockAuthenticationMethodService, MockAuthenticationService, MockUserResourceService, TestConfigurationService,
+    TranslateLibModule, UserResourceService,
+    WrappedBoolean
+} from '@netgrif/components-core';
+import {AngularResizedEventModule} from 'angular-resize-event';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+import {HttpClientTestingModule} from '@angular/common/http/testing';
 
 describe('I18nDividerFieldComponent', () => {
     let component: I18nDividerFieldComponent;
     let fixture: ComponentFixture<TestWrapperComponent>;
 
     beforeEach(async () => {
         await TestBed.configureTestingModule({
-            declarations: [I18nDividerFieldComponent, TestWrapperComponent]
-        })
-            .compileComponents();
-    });
-
-    beforeEach(() => {
+            imports: [
+                MaterialModule,
+                AngularResizedEventModule,
+                BrowserAnimationsModule,
+                TranslateLibModule,
+                HttpClientTestingModule
+            ],
+            providers: [
+                {provide: AuthenticationMethodService, useClass: MockAuthenticationMethodService},
+                {provide: ConfigurationService, useClass: TestConfigurationService},
+                {provide: AuthenticationService, useClass: MockAuthenticationService},
+                {provide: UserResourceService, useClass: MockUserResourceService}
+            ],
+            declarations: [I18nDividerFieldComponent, TestWrapperComponent],
+            schemas: [CUSTOM_ELEMENTS_SCHEMA]
```

Comment:
why is this necessary?

Reply:
CUSTOM_ELEMENT_SCHEMA is used in each dataField test so I included it in these two tests too.
Maybe @Kovy95 knows more about it.

Reply:
its okay, i think mostly its used in datafields to get rid of some warnings in console during tests, that you using some custom elements in html